src/applications/hca/tests/unit/utils/helpers/form-config.unit.spec.js

```diff
@@ -626,4 +628,46 @@ describe('hca form config helpers', () => {
       expect(useJsonFacilityList(formData)).to.be.true;
     });
   });
+
+  context('when `insuranceTextOverrides` executes', () => {
+    const defaultOutput = {
+      cancelAddTitle: content['insurance-info--array-cancel-add-title'],
+      cancelEditTitle: content['insurance-info--array-cancel-edit-title'],
+      cancelEditDescription:
+        content['insurance-info--array-cancel-edit-description'],
+      cancelEditReviewDescription:
+        content['insurance-info--array-cancel-edit-review-description'],
+      cancelAddYes: content['insurance-info--array-cancel-add-yes'],
+      cancelEditYes: content['insurance-info--array-cancel-edit-yes'],
+    };
+    const executeMethod = ({ item, output }) => {
```

Comment:
🎨 Might be nice for this to be a bit more specifically named? `expectMatchingInsuranceText` or something?

eh, it's pretty isolated though - not a blocker.